lib/extensions/zigzag_line_to_satin.py

```diff
@@ -50,8 +50,6 @@ def effect(self):
 
             element.node.set('d', " ".join(d))
             element.set_param('satin_column', True)
-            # remove dashes
-            element.update_dash(False)
```

Comment:
I think we still need this?

Reply:
At the moment yes, but not if we go forward with our idea. I'm a bit confused what to do here and what to do later :D
But the update_dash method I removed entirely when I removed the dash modifications, so either we leave it all in or just remove it at this point I suppose.